config/module.config.php

```diff
@@ -14,6 +14,8 @@
     ],
     'cli_commands' => [
         'factories' => [
+            \ConferenceTools\Checkin\Cli\Command\GenerateQR::class =>
```

Comment:
Add a `use ConferenceTools\Checkin\Cli\Command` and then use `Command\GenerateQR::class` and `Command\GenerateQRFactory` ?